doc/nrf/ug_nrf_cloud.rst

```diff
@@ -56,6 +56,54 @@ MQTT overview
 * Once connected, the device subscribes to the desired MQTT topics.
 * Each MQTT publish event contains the MQTT topic and the payload.
 
+Security
+********
+
+A device can successfully connect to `nRF Cloud`_ using REST if:
+
+* the device contains a correct x509 CA certificate, and private key
+* the public key derived from the private key is registered with an nRF Cloud account
+* the device calls nRF Cloud REST APIs which require a JSON Web Token (JWT) by providing a JWT signed by the private key
+
+A device can successfully connect to `nRF Cloud`_ using MQTT if:
+
+* the device contains a correct x509 CA certificate, device certificate, and private key
+* the device id and device certificate are provisioned with nRF Cloud
+* the device id is associated with an nRF Cloud account
+
+`nRF Cloud`_ supports two different ways for creating and installing these certificates both in the device and the cloud.
+See `nRF Cloud Provisioning`_ for more information.
+
+* Online
+  In the user's nRF Cloud account, the user enters the device id in a web form, then downloads a JSON file containing the CA certificate, device certificate, and private key.
+  Alternatively, the user can access the nRF Cloud REST API to do so.
+  The user programs the credentials in the JSON file into the device using LTE Link Monitor.
+  See :ref:`nrf9160_ug_updating_cloud_certificate` for details.
+
+* Offline
```

Comment:
Is "Offline" provisioning used anywhere else?  Why not just say this is Preconnect provisioning?
Also, its only "offline" for the first few steps.